types/braintree/index.d.ts

```diff
@@ -816,6 +827,55 @@ declare namespace braintree {
         | 'VisaCheckoutCard'
         | 'SamsungPayCard';
 
+    /**
+     * Webhooks
+     */
+
+    export class SampleNotification {
+        bt_signature: string;
+        bt_payload: string;
+    }
+
+    export class WebhookNotification {
```

Comment:
Looks like `timestamp` is part of the WebhookNotification as well, can we add it here? https://developers.braintreepayments.com/reference/general/webhooks/overview